docs/references/react/use-organization.mdx

```diff
@@ -28,13 +28,29 @@ const {
 To keep network usage to a minimum but to also provide a smooth auto-updating experience to developers, we made the `invitationList` attribute available only when its respective parameter is present when calling the hook.
 
 ```jsx
-const { invitationList } = useOrganization();
-// invitationList is undefined.
+const { invitations } = useOrganization();
+// invitationList.data will never be populated
```

Comment:
Is this comment even necessary to keep?

Reply:
i believe yes, because it indicates that you need to opt-in in order to fetch data.

Reply:
right, sorry I should have been more clear. should it be `invitations.data` instead of `invitationsList.data` because `invitationsList` is now deprecated in favour of `invitations`?

Reply:
Oh i see, yeap i missed that one 😅